packages/website/src/features/Deploy/QueueFromGitOpsPage.tsx

```diff
@@ -562,8 +554,7 @@ export default function QueueFromGitOps() {
     (onChainPrevPkgQuery.isFetched &&
       !prevDeployLocation &&
       tomlRequiresPrevPackage &&
-      !previousPackageInput) ||
-    canTomlBeDeployedUsingWebsite;
+      !previousPackageInput);
```

Comment:
🐛 This change fixes a bug were the "Preview Transactions Button" was disabled if the package was new.